src/Grpc.AspNetCore.Server/InterceptorRegistration.cs

```diff
@@ -17,6 +17,7 @@
 #endregion
 
 using System;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Grpc.AspNetCore.Server
```

Comment:
Did we want to expose args? I see that as a item of the linked issue.

Reply:
Future PR